### DISCUSSION TITLE
Sprint project candidates

### DISCUSSION THREAD

**Discussion by tai271828** [solvcon/devenv#42] [General (discussions)]
It would be great if we have a collection of sprint project candidates, so we can have consensus to prioritize tasks efficiently. Please append the project candidates anytime as your wish.

- Extend unittest based on shunit2. Ref https://github.com/solvcon/devenv/pull/40

**Comment by yungyuc** [comment]
@j8xixo12 I am thinking maybe this discussion thread can be used for preparing for the next sprint(s).  Do you have a plan?

**Reply by j8xixo12** [reply]
No I don't have plan for now, so you mean if anyone have some idea of sprint project, they can put their idea to this thread right?
That's a good idea!

**Reply by yungyuc** [reply]
Yes, that's what we meant.

Further, you can add the link to the discussion thread in the sprint web page.  People visiting the sprint page can come to the thread and discuss about what they want.

It depends on how early you set up the page for the next sprint.  The earlier the more discussions.

**Reply by j8xixo12** [reply]
I start to create a new sprint web page, maybe we can based on #40 added more test case
but it doesn't looks attractive :P

**Reply by tai271828** [reply]
It sounds attractive to me : D

**Reply by yungyuc** [reply]
It sounds good to me too!
